docs/tags/custom-tags.md

```diff
@@ -0,0 +1,50 @@
+title: Custom Tags 
+description: A Tag is an attribute of a data point (or metric) that could be used for grouping and filtering. The Sematext Agents support configuring custom tags. They can be specified in the Agent's configuration files. For example, tags like `env:prod` can be configured on all production servers.
+
+The Sematext Agents support configuring custom tags. They can be specified in the Agent's configuration files. For example, tags like `env:prod` can be configured on all production servers and `env:dev` on all development servers. The data can then be filtered in the UI using these tags. The maximum allowed length for both name and value is 1024 characters. They are mapped to `os.host` tag by the Agent. Configured tags are optional and can be changed at any time. The name and value of custom tags should match this regex: `[a-zA-Z0-9_\-=\+\.]*`.
```

Comment:
I wonder if we should have Tags > Specs page where we show the list of tag specs/limits.  Like max length, allowed characters, max number of tax per ____, something about case sensitivity, etc.

Reply:
I'm all for it, but don't know what to write in such a page. Who would know what exactly we should add to the Tags > Specs page?

Reply:
I've created a task to not lose track of it: https://sematext.atlassian.net/browse/SC-6875

Reply:
> Who would know what exactly we should add to the Tags > Specs page?

You should.  Bojan would.  I might.
Let's just do it now.  It's 3.5 lines of text.  Stick there what you know (it's stuff like what you can see above - max length, allowed chars), and we can expand from there if there is more to add.